src/Compilers/CSharp/Test/Emit/Emit/DynamicAnalysis/DynamicInstrumentationTests.cs

```diff
@@ -2599,7 +2599,7 @@ class C
 {
     int x = 1;
 
-    static C() { }
+    static C() { Console.WriteLine(); }
```

Comment:
>Console.WriteLine(); [](start = 17, length = 20)

What is the purpose of this change? And below as well.

Reply:
artifact of the approach which excluded empty user-defined static constructors from emit. reverting.